sample-app/redisearch-jedis-rest/README.md

```diff
@@ -28,13 +28,13 @@ This command will create a new image and build the maven project into it.
 
 ```shell script
 > docker run --rm  \
-     --env "REDIS_URL=redis://host.docker.internal:6379" \
+     --env "REDIS_URL=redis://redis-stack:6379" \
```

Comment:
May need to be updated due to compose. In compose the current names are redisearch.

Reply:
this is already showing changed on my repository so not sure about this...